src/ui/DidYouMean/DidYouMean.ts

```diff
@@ -142,7 +142,8 @@ export class DidYouMean extends Component {
       const correctedSentence = this.buildCorrectedSentence(results.queryCorrections[0]);
       this.correctedTerm = results.queryCorrections[0].correctedQuery;
 
-      const correctedWordEl = $$('a', {}, correctedSentence).el;
+      const correctedWordEl = $$('a', { href: 'javascript:void(0);' }, correctedSentence).el;
+      correctedWordEl.tabIndex = 0;
```

Comment:
We could add the `tabindex` attribute when creating the object on line 145 rather than appending it here. e.g.

```
const correctedWordEl = $$('a', { href: 'javascript:void(0);', tabindex: 0 }, correctedSentence).el;
```

Reply:
Good point.

Since I changed the element to a button, I'll remove the tabindex altogether.